spirv_glsl.cpp

```diff
@@ -6162,6 +6162,10 @@ std::string CompilerGLSL::to_texture_op(const Instruction &i, bool sparse, bool
 		opt = &ops[5];
 		length -= 5;
 		gather = true;
+		if (options.es && options.version < 310)
+			SPIRV_CROSS_THROW("textureGather requires ESSL 310.");
+		else if (!options.es && options.version < 400)
```

Comment:
Don't you have to check for Dref opcode here?

Reply:
Not sure what you mean; this bit is right under
```cpp
	case OpImageDrefGather:
	case OpImageSparseDrefGather:
```

Reply:
Ah, right. Derp.